rpm/inicpp.spec

```diff
@@ -0,0 +1,48 @@
+Name: inicpp
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: C++ parser of INI files.
+License: MIT
+BuildRequires: cmake, make, (gcc-c++ or clang)
+URL: https://github.com/SemaiCZE/%{name}
+Patch0: %{name}-cmake.patch
+Source0: https://github.com/SemaiCZE/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+%description
+Modern, C++14 library for parsing INI files with schema validation.
+
+%prep
+%autosetup
+
+%build
```

Comment:
What about use in-tree build here? In my opinion it's allright for single purpose builds and you need just 2 lines (instead of 5).

Reply:
Respectfully, I disagree.  I prefer the rpm to be built the same way as when I manually build the project to avoid unforeseen errors.  It's your project, so feel free to change it.